kernel/Dockerfile.zfs

```diff
@@ -53,7 +53,7 @@ RUN ./autogen.sh && \
     make install
 
 # Run depmod against the new module directory.
-RUN depmod -ae `ls /lib/modules`
+RUN cd /lib/modules ; depmod -ae * 
```

Comment:
I would make that
```
RUN cd /lib/modules && \
    depmod -ae *
```
It's unlikely to fail, but is more in line with the general style we use.